### PR TITLE
add subscription date

### PR DESCRIPTION
**Pull request by DonDuala**

### CODE REVIEW

File: src/gso.ts [hunk 1]
```diff
@@ -165,7 +166,7 @@ export class GSO {
       new BN(1), /* period_num */
       new BN(lockupRatioPerMillionLots),
       new BN(optionExpiration),
-      new BN(optionExpiration), /* subscription_period_end */
+      new BN(subscriptionExpiration), /* subscription_period_end */
```
Comment (brittcyr) [inline src/gso.ts:169]: comment no longer needed now that the variable is descriptive

File: src/gso.ts [hunk 2]
```diff
@@ -132,6 +132,7 @@ export class GSO {
   public async createConfigInstruction(
     lockupRatioPerMillionLots: number,
     optionExpiration: number,
+    subscriptionExpiration: number,
```
Comment (brittcyr) [inline src/gso.ts:135]: subscription_period_end is the name of the variable in the program. Can we keep the naming consistent?

**Review by brittcyr — APPROVED** [review]

File: src/gso.ts (hunk 2 above)
Comment (DonDuala) [reply src/gso.ts:135]: complains about camel case with this variable naming

Comment (DonDuala) [reply src/gso.ts]: renamed subscriptionPeriodEnd

**Pull request merged**